### PR TITLE
Fix indicator image of checkable QGroupBox for checked/unchecked stat…

### PR DESCRIPTION
**Pull request by sawt00th**
Fix problem with QGroupBox indicator #93
groupbox_fix

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - 2.5.3
   - Add future warning and pending deprecation for 3.0 version preparation #89
   - Add ISSUE_TEMPLATE to ask for default information on issue tracker
+  - fix #93 issue
```
Comment (ColinDuquesnoy) [inline CHANGES.md]: @dpizetta released 2.5.3 a few days ago. 

I think this change should at least be in a new ``2.5.4`` section. @dpizetta What do you think?

**Comment by ColinDuquesnoy** [comment]
Thank you for your contribution! I have just one nitpick with the change you made to the changelog (see the inline comment).

**Comment by ColinDuquesnoy** [comment]
@sawt00th I'd wait for @dpizetta answer. That depends on his plans for next releases.

File: CHANGES.md (hunk 1 above)
Comment (dpizetta) [reply CHANGES.md]: @ColinDuquesnoy, @sawt00th I agree. Please, insert a new version 2.5.4 with this fix, remember to change the version itself in __init__.  I would ask you to add more details to your message in changelog with something like this: Fix indicator image of checkable QGroupBox, #93, so if you read it you don't need to search by the issue.

BTW, thank you!

**Comment by sawt00th** [comment]
ok, i will do it.
its my first pull request, and i have a question.
Should i commit change and push on GitHub without new pull request, or after that create new pull request?

**Comment by dpizetta** [comment]
Hi, if you change your pull request, i.e, commit your changes to your rep, it will be automatically included to pull request you have already created. Easy peasy :)

**Pull request merged**